### PR TITLE
Tools [WIP]

### PR DESCRIPTION
**Pull request by HoldYourWaffle**
So I was done with my big-crawler refactor, and figured I could go working on this =)

(I'll be adding more commits to this today [it's 10AM here now])

(it may be handy to squash merge this cause else you get all kind of "Merge remote-blablabla" commits in your history)

### CODE REVIEW

**Comment by HoldYourWaffle** [comment]
Nice we got almost the same coding style =)

**Comment by HoldYourWaffle** [comment]
Should I just add Andesite/Diorite/Granite tools or more variants?
(haven't done coloring yet at all)

File: src/main/resources/mcmod.info [hunk 1]
```diff
@@ -7,7 +7,7 @@
   "mcversion": "${mcversion}",
   "url": "",
   "updateUrl": "",
-  "authorList": ["Cruz Bishop, Ravi van Rooijen"],
+  "authorList": ["Cruz Bishop, ShadowCoder69"],
```
Comment (CruzBishop) [inline src/main/resources/mcmod.info:10]: Fair enough, I'll cherry pick these in to develop :)

**Comment by CruzBishop** [comment]
Thanks for starting work on this :) I'll cherry pick the two commits you started off with so far and go from there

It's up to you, I was going to work on a little script to make dyed blocks (and I guess items) a lot easier to generate when I get some time off next week, so just diorite/andesite/granite should be okay for now :+1: 

If anything major comes up, I'll send you a pull request - but from what I've seen so far, you'll do good :)

**Comment by HoldYourWaffle** [comment]
I'll do diorite/andesite/granite for now, and do the dyed stuff when your script is done :)

**Comment by CruzBishop** [comment]
Sounds good, thank you :)

**Comment by CruzBishop** [comment]
Sorry if you get spammed by notifications in a minute, I'm going to try something on here and I'm not sure if it will work

**Comment by HoldYourWaffle** [comment]
No problem. Was just going to lunch.

**Comment by CruzBishop** [comment]
Mmm, lunch. You're making me hungry again :cake: 

I think I just sent you the right access request, then if you accept you'll be able to pull/push directly from here and End Deco if you want (otherwise it might make it easier to do a hard reset to develop)

There's another way to do it as well, but it's easier this way :)

Enjoy your lunch, I might go find something to chew on then come back and play around a bit

**Comment by HoldYourWaffle** [comment]
Back again, joined Techern.
I can just create a feature branch now right?
(never worked in a team before)

**Comment by CruzBishop** [comment]
Thank you :)

Yeah, you should be right to pull develop and start a feature branch now if you want :+1:

**Comment by HoldYourWaffle** [comment]
I'm gonna try something so this PR needs to be closed.

**Comment by CruzBishop** [comment]
Alright, good luck :)

I just merged the first batch of pressure plates into develop so sorry if that messed anything up

**Comment by HoldYourWaffle** [comment]
That's part of the reason why i'm trying this ;)

I don't mind tho, I'm not very experienced with git(hub) so still need to learn alot

**Comment by CruzBishop** [comment]
:)

All good, the great thing about git is that if something goes wrong, somebody somewhere will have a history of a working copy :page_facing_up:

**Comment by HoldYourWaffle** [comment]
Try-thingy succeeded =)

Now I have my crawler & IE in 1 workspace

**Comment by CruzBishop** [comment]
That's great :grinning:

Should make it easier, thank you again for helping :)

**Comment by HoldYourWaffle** [comment]
I like this project (played IA already) and I like opensource.
Good combi isn't it?

**Comment by CruzBishop** [comment]
Yep, it sure is :)

I'm about to start on slabs again but I also just took some painkillers for my toes, if I fall asleep have a great afternoon / night - One more day of work then two off :+1:

**Comment by HoldYourWaffle** [comment]
Haha :smile: 

Btw what version number schould I use in the since tags for my feature?

**Comment by CruzBishop** [comment]
Oops, I tagged it as 0.9 :p I'll change it in a sec

Should be right to do @since 0.0.2

**Pull request closed without merging**